integration/fixtures/tracing/tempo.yaml

```diff
@@ -0,0 +1,58 @@
+server:
+  http_listen_port: 3200
+
+query_frontend:
+  search:
+    duration_slo: 5s
+    throughput_bytes_slo: 1.073741824e+09
+  trace_by_id:
+    duration_slo: 5s
+
+distributor:
+  receivers:
+    zipkin:
+    otlp:
+      protocols:
+        http:
+        grpc:
+
+ingester:
+  lifecycler:
+    address: 127.0.0.1
+    ring:
+      kvstore:
+        store: inmemory
+      replication_factor: 1
+    final_sleep: 0s
+  trace_idle_period: 1s
+  max_block_duration: 1m
+  complete_block_timeout: 5s
+  flush_check_period: 1s
+compactor:
+  compaction:
+    block_retention: 1h
+
+metrics_generator:
+  registry:
+    external_labels:
+      source: tempo
+      cluster: docker-compose
+  storage:
+    path: /tmp/tempo/generator/wal
+    remote_write:
+      - url: http://prometheus:9090/api/v1/write
+        send_exemplars: true
+
+storage:
+  trace:
+    backend: local                     # backend configuration to use
+    wal:
+      path: /tmp/tempo/wal             # where to store the the wal locally
+    local:
+      path: /tmp/tempo/blocks
+
+overrides:
```

Comment:
rather than this have you considered giving an example with OTEL-Collector instead for example

```
receivers:
  otlp:
    protocols:
      grpc:
      http:
exporters:
  otlp/tempo:
    endpoint: tempo:4317
  prometheus/otlp:
    endpoint: "0.0.0.0:8080"
connectors:
  servicegraph:
  spanmetrics:
service:
  pipelines:
    traces:
      receivers: [otlp]
      exporters: [otlp/tempo,spanmetrics,servicegraph]
    metrics
      receivers: [spanmetrics,servicegraph]
      exporters: [prometheus/otlp]
```

Reply:
Thanks for the suggestion @trajano, we indeed introduced a collector in between Traefik and Tempo in integration tests!